### ISSUE TITLE
zte mf823 support

### ISSUE THREAD

**Issue by hryamzik** [gammu/gammu#310]
This modem is presented in system as a network device and but can be switched to diagnostic mode to act as an android (!) device.

Any chance to get it running? Might be related to #81.

**Comment by nijel** [comment]
If it can be switched to understand AT commands, it will most likely work. I've quickly Googled, and it seems to be possible by switching to to factory mode (see https://wiki.archlinux.org/index.php/ZTE_MF_823_(Megafon_M100-3)_4G_Modem).

**Comment by hryamzik** [comment]
Yep but it fails on listing SMS (tried just a single AT command). So factory mode is for different settings only.

**Comment by nijel** [comment]
Probably it doesn't implement required AT commands, it's hard to tell without documentation...

**Comment by hryamzik** [comment]
I think it doesn't but it can act as an adb device. Another option is to access eb interface but that seem to be overcomplicated.

**Comment by nijel** [comment]
We currently have no support for Android devices, see #81.

**Comment by hryamzik** [comment]
I see, I've referenced this issue in initial comment.
